app/views/clients/show.html.erb

```diff
@@ -0,0 +1,6 @@
+<!DOCTYPE html>
+<%= render "homepage/header" %>
```

Comment:
what is this doing here? should it not be in the layout?

Reply:
fixed.